MailManager/src/main/java/inc/softserve/resources/EnvelopeResource.java

```diff
@@ -51,10 +53,13 @@ public long saveEnvelope(@QueryParam("Subject") String subject,
     }
 
     @GET
+    @Timed
     public EnvelopeState getEnvelopeStateById(@QueryParam("id") long id){
         Envelope envelope = dao.getById(id);
-        if(envelope != null)
+        if(envelope != null) {
+            System.out.println("Method ended");

```

Comment:
I believe that we don't need this in production code. You may introduce logging and move that output into DEBUG logging level, for example.